backends/client.go

```diff
@@ -4,6 +4,7 @@ type Client interface {
 	Load() ([]byte, error)
 	Save([]byte) error
 	Backup() error
+	SafeLocation() string
```

Comment:
A remote backend (http, ssh, etc) is indeed what I had in mind. Reviewing the code again, I see your point and agree. 👍